consensus/parlia/snapshot.go

```diff
@@ -248,10 +283,10 @@ func (s *Snapshot) apply(headers []*types.Header, chain consensus.ChainHeaderRea
 	for _, header := range headers {
 		number := header.Number.Uint64()
 		// Delete the oldest validator from the recent list to allow it signing again
-		if limit := uint64(len(snap.Validators)/2 + 1); number >= limit {
+		if limit := snap.minerHistoryCheckLen() + 1; number >= limit { // `+1` for genesis block which has no miner
```

Comment:
`// `+1` for genesis block which has no miner`
what does it mean?

Reply:
the comment is not right. I will delete it